crates/sui-transactional-test-runner/src/test_adapter.rs

```diff
@@ -958,6 +1054,55 @@ impl<'a> SuiTestAdapter<'a> {
             Some(fake) => format!("fake({})", fake),
         }
     }
+
+    fn compile_source_units(
```

Comment:
Copied over from `move-lang/move` changes: https://github.com/move-language/move/pull/1017/files

Reply:
I have made this public and refactored a lot of things if we bump the move version

Reply:
awesome, I've based this off https://github.com/MystenLabs/sui/pull/9932 now to pick up those changes and fixed up this PR to use them